src/Apps/Artwork/Components/ArtworkPageBanner.tsx

```diff
@@ -87,9 +108,83 @@ const UnpublishedArtworkBanner = () => (
   </FullBleedBanner>
 )
 
+interface OrderBannerProps {
+  order: NonNullable<
+    NonNullable<
+      NonNullable<
+        NonNullable<
+          NonNullable<
+            ArtworkPageBannerOrdersQuery$data["me"]
+          >["ordersConnection"]
+        >["edges"]
+      >[number]
+    >["node"]
+  >
+}
+
+const OrderBanner: FC<React.PropsWithChildren<OrderBannerProps>> = ({
+  order,
+}) => {
+  let variant: BannerVariant = "defaultLight"
+  let copy = "You ordered this artwork"
+  let linkCopy = "View order"
+
+  switch (order.buyerState) {
+    case "APPROVED":
+      copy = "Your order has been approved!"
+      variant = "success"
+      break
+    case "COMPLETED":
+      copy = "Your order is complete!"
+      break
+    case "OFFER_RECEIVED":
+      copy = "The gallery has responded to your offer."
```

Comment:
Like this for example is currently is `Counteroffer received`. I feel like we should be consistent with those. We can also see if `actionPrompt` exists in this `displayTexts` and based on that link to a different page `counteroffer` specifically.